### ISSUE TITLE
Accessibility fails on gcIntranet due to contrast ratio

### ISSUE THREAD

**Issue by StdGit** [wet-boew/cdts-sgdc#150]
The intranet teams are reporting accessibility fails originating from the GC template in our CDTS  – looks like it is the contrast ratio to the departmental links in the header

**Comment by ahmad-shahid** [comment]
This is being tracked in #195. Closing this issue.
